example-server.js

```diff
@@ -0,0 +1,48 @@
+var http = require('http')
+var extend = require('xtend')
+var vdom = require('virtual-dom')
+var createApp = require('./index')
+var h = vdom.h
+
+/*
+* create the app passing the container element and virtual-dom
+*/
+var app = createApp(vdom)
+
+/*
+* The only way to modify state is to trigger an action
+* the modifer function is where you change state based on the type of an action
+*/
+function modifier (action, state) {
```

Comment:
The `modifier` function is almost moot with server-side rendering – at least it seems that way. Are there situations where the modifier function will need to do a lot on the server? I'm not sure. What if we could omit it?

```js
var render = app.start({
  example: false,
  title: 'My web page title'
})
```

And if the function isn't available just provide `initialState`. I'm not sure if that's weird or not.

Anyway, we could at least simplify this example so `modifier` just returns `state` if it won't really be doing anything.